concourse/deploy-cf-perftest.yml

```diff
@@ -385,6 +384,46 @@ jobs:
           params:
             alert_type: failed
 
+  - name: bosh-clean-up
+    serial: true
+    plan:
+      - get: cf-deployment-concourse-tasks
+      - get: bbl-state
```

Comment:
bbl-state doesn't changed in this job, so `bosh-clean-up` job will not automatically triggered.
we advise, add `cf-deployment` in get, which means, if there is a new cf-deployment, `bosh-clean-up` job will run after `bosh-deploy`, like:
` - get: cf-deployment
    passed:
      - bosh-deploy
    trigger: true
`